lib/install.js

```diff
@@ -282,7 +282,15 @@ function install(name, target, options, seen) {
       if (options.inject)
         return pkg.inject(resolution, downloadDeps);
 
-      return pkg.download(resolution, options.override, options.unlink, downloadDeps);
+      var override;
+
+      try {
+        override = options.override || config.pjson.originalPjson.jspm.overrides[name];
+      } catch(e) {}
+
+      // TODO: compound the options.override onto the package.json override?
```

Comment:
Thought I'd ask about it since you're doing that with the registry overrides right?